From fdccb5e5085f4c8286e9ce993079386347019637 Mon Sep 17 00:00:00 2001
From: Matthieu Herrb <matthieu.herrb@laas.fr>
Date: Fri, 23 May 2014 17:05:44 +0200
Subject: [PATCH] testLib: only print error message if there was an error.

Needed since h2getErrMsg() was changed to display 'Unknown error'
instead of 'OK'  when errno is 0.
---
 lib/testLib.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git lib/testLib.c lib/testLib.c
index 622e6c0..15346f7 100644
--- lib/testLib.c
+++ lib/testLib.c
@@ -674,7 +674,8 @@ testStopModule(TEST_STR *eid)
 			     (void *) NULL, 0, (FUNCPTR) NULL);
   
   bilan = errnoGet();
-  h2perror("Final reply of request Abort");
+  if (bilan != OK) 
+	  h2perror("Final reply of request Abort");
 
   /* Requete d'abort DU MODULE qui a reussi */
   if (status == FINAL_REPLY_OK && bilan == OK) 
@@ -719,7 +720,8 @@ testAbortRqst(TEST_STR *eid)
 			     (void *) NULL, 0, (FUNCPTR) NULL);
   
   bilan = errnoGet();
-  h2perror("Final reply of request Abort");
+  if (bilan != OK) 
+	  h2perror("Final reply of request Abort");
 
   /* Requete d'abort DU MODULE qui a reussi */
   if (activity == GENOM_END_MODULE && status == FINAL_REPLY_OK && bilan == OK) 
-- 
1.7.9.5

